src/rust/gen-compile-cache/cxx-bridge.c++

```diff
@@ -36,6 +37,7 @@ constexpr v8::ScriptCompiler::CompileOptions compileOptions = v8::ScriptCompiler
   auto data = ccIsolate.runInLockScope([&](CompileCacheIsolate::Lock& isolateLock) {
     return JSG_WITHIN_CONTEXT_SCOPE(isolateLock,
         isolateLock.newContext<CompilerCacheContext>().getHandle(isolateLock), [&](jsg::Lock& js) {
+      v8::TryCatch tryCatch(js.v8Isolate);
```

Comment:
Wrapping in `js.tryCatch(...)` would be better here as it will handle the fatal / can't continue cases better.